code/game/machinery/vending_vr.dm

```diff
@@ -1205,7 +1205,7 @@
 					/obj/item/clothing/mask/gas/plaguedoctor/fluff = 600,
 					/obj/item/clothing/under/owl = 400,
 					/obj/item/clothing/mask/gas/owl_mask = 400,
-					/obj/item/clothing/under/waiter = 200,
+					/obj/item/clothing/under/waiter = 50,
```

Comment:
Should be 100

Reply:
Uniform item, so i think it should be 50 IMO